tmva/tmva/inc/TMVA/DNN/Minimizers.h

```diff
@@ -76,8 +74,14 @@ class TGradientDescent
    TGradientDescent(Scalar_t learningRate,
                     size_t   convergenceSteps,
                     size_t   testInterval);
-   /** Reset minimizer object to initial state. Does nothing for this minimizer. */
-   void Reset() {};
+
+   /** Reset minimizer object to default state. */
+   void Reset()
+   {
+      fMinimumError = 1e100;
```

Comment:
Look - I'm just commenting at what I'm seeing, I'm not saying that a) you should fix it nor b) that it should be part of this PR. I let Lorenzo and you deal with that :-)